6/6.css

```diff
@@ -0,0 +1,39 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+.blacksmile {
+  width: 200px;
+  height: 200px;
+  margin-bottom: 30px;
+}
+.fa-smile {
```

Comment:
Try to omit as much as possible styling for third party library classes.
It means leave .fa-smilte class, untouched and try to style this smile by creating your own class like black-smile.
Why we shouldn't change styling for .fa-smile class?